core/src/main/scala/org/apache/spark/deploy/master/ExecutorDesc.scala

```diff
@@ -27,6 +27,9 @@ private[master] class ExecutorDesc(
     val memory: Int) {
 
   var state = ExecutorState.LAUNCHING
+  // resources(e.f. gpu/fpga) allocated to this driver
+  // map from resource name to its addresses
+  private var _resources: Map[String, Seq[String]] = _
```

Comment:
should use something like ResourceAllocation/ResourceInformation/ExecutorResourceInfo instead of Seq[String], depending on how its used later

Reply:
Sure, will replace those variables later.